.formatter.exs

```diff
@@ -0,0 +1,4 @@
+[
+  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"],
+  line_length: 120
```

Comment:
Is 120 characters a community standard? If not, what would you think about using a 100 character limit? I still get some side scrolling with 120 character width lines in my editor. I like the auto-formatting for style though :+1:

Reply:
No, 120 in not a standard but it looks OK with 15" MacBook screen :) Anyway it looks like default value is 98 characters:   
https://hexdocs.pm/elixir/Code.html#format_string!/2
I will remove this setting and run `mix format` again!